lib/TelegramHandlers/TgMessageHandler.js

```diff
@@ -31,14 +31,23 @@ class TgMessageHandler {
      * @param {string} userMessage - The message the user sent that we want to relay.
      */
     RelayMessage(from, userMessage) {
-        if (!this.Enabled) {
+        const self = this;
+        if (!self.Enabled) {
             return;
         }
 
         let username = TgHelpers.ResolveUserName(from);
-        let message = this._prefixSuffixConfig.prefix + username + this._prefixSuffixConfig.suffix + " " + userMessage;
-
-        this._action(message);
+        const messagePrefix = self._ircConfig.prefix + username + self._ircConfig.suffix + " ";
+        const messageSplitRegex = new RegExp(`.\{1,${self._ircConfig.maxMessageLength - messagePrefix.length}}`, 'g');
+
+        userMessage.toString().split(/\r?\n/).filter(function(line) {
```

Comment:
Can you add a comment here just explaining that you are splitting the string?  If someone new looks at this code, it may not be obvious what is going on.

Reply:
@xforever1313 is a comment super necessary here? It *is* using the `.split` method, so the string splitting should be inherently known?